lib/plugin/security/authorization.js

```diff
@@ -4,7 +4,7 @@ const _ = require('lodash')
 const Boom = require('boom')
 const async = require('async')
 
-function buildAuthorization (udaru, config) {
+function buildAuthorization (config) {
```

Comment:
shouldn't we also remove config here and use the decorated one?

Reply:
:+1: As the PR got big I'll merge it then see what enhancements we can come up with.